docs/kratos/social-signin/30_apple.mdx

```diff
@@ -88,6 +88,8 @@ Follow these steps to add Apple as a social sign-in provider to your project usi
          // Therefore we only return the email if it (a) exists and (b) is marked verified
          // by Apple.
          [if 'email' in claims && claims.email_verified then 'email' else null]: claims.email,
+         [if 'given_name' in claims then 'firstName' else null]: claims.given_name,
+         [if 'last_name' in claims then 'lastName' else null]: claims.last_name,
```

Comment:
This only works if the identity schema accepts "firstName", which the default (preset://email) doesn't, maybe we need to make that clear here? E.g. with multiple tabs?